keyboards/dztech/dz60rgb/keymaps/default/keymap.c

```diff
@@ -136,20 +113,15 @@ void rgb_matrix_indicators_user(void) {
           rgb_matrix_set_color(62, 0x00, 0x00, 0x00);
           break;
 	      case _LAYER4:
```

Comment:
Strange indentation

Reply:
@drashna yes, I've already tested, rgb_light will be off when close the PC. 
but there is a strange bug, it works fine on my notebook, but on my PC, I can't press any key if the PC is shutting down, or it will not be recognized after powered on next time.